src/app/lib/config/services/index.js

```diff
@@ -15,6 +15,7 @@ export default {
     brandName: 'Default Brand Name',
     externalLinkText: ', default external link suffix',
     imageCaptionOffscreenText: 'Default image caption prefix, ',
+    videoCaptionOffscreenText: 'Default video caption prefix, ',
```

Comment:
Great that you've added this here. There is also a `services/news.js` and a `services/persian.js` for which we'd want to add the `videoCaptionOffscreenText`.